CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributions Welcome
+
+First off, thank you for considering contributing to the Github-Webhook-Resource! This resource is a simple project, but has plenty of room for improvement. Many features of Github's webhook implementation are yet to be built into the Github-Webhook-Resource, and all contributions are welcome.
+
+If you're just looking for quick feedback for an idea or proposal, feel free to open an
+[issue](https://github.com/homedepot/github-webhook-resource/issues).
```

Comment:
How about linking this to the 'create new issue' page instead?  https://github.com/homedepot/github-webhook-resource/issues/new